tests/test_data_extraction.py

```diff
@@ -82,3 +82,10 @@ def test_data_extractor_all_modes(sim):
         extraction_method="panorama",
     )
     assert len(extractor) > 0
+
+
+def test_triangle_id(sim):
+    ## test that empty frames can be rendered without a scene mesh
+    obs = sim.step("move_forward")
+    triangle = obs["triangle_sensor"]
```

Comment:
you can freeze average value of `obs["triangle_sensor"]`

Reply:
Do you mean save the average value of the sensor at some state s0, then during testing set the state to s0 again and see if we get the same average value of the sensor?

Reply:
Just do `np.avg(triangle) == TEST_AVG_TRIANGLE_VALUE` in the test. Then test will fail output of the sensor will change because of some code changes. Then we will know that all functionality depend on triangles sensor will be affected.